src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -265,3 +268,39 @@ When migrating from our traditional Azure integrations to the Azure Monitor inte
 
 * When you enable an Azure Monitor integration, new separate entities will be created for all of your resources. The entities created by the Azure Polling integration are staying as they are. This means you must update dashboards, alerts, and any other capability that refrences those entities.
 * Old entities are available for 24 hours.
+
```

Comment:
Starting here, and to the end of the PR: these are the new steps added.
Please review carefully.